src/tp322.jl

```diff
@@ -0,0 +1,25 @@
+# TP problem 322 in NLS format
+#
+#   Source:
+#   Klaus Schittkowski,
+#   More test examples for nonlinear programming codes,
+#   Lecture Notes in Economics and Mathematical Systems 282,
+#   Springer Verlag Berlin Heidelberg, 1987
+#   10.1007/978-3-642-61582-5
+#
+# A. Montoison, Montréal, 06/2018.
+
+export tp322
+
+"Test problem 322 in NLS format"
+function tp322()
+
+  nls = Model()
+  @variable(nls, x[i=1:2], start=0)
+
+  @NLexpression(nls, F1, x[1] - 20)
+  @NLexpression(nls, F2, x[2] + 20)
+
+  @NLconstraint(nls, x[1]^2/100 + x[2]^2/0.01 - 1 == 0)
```

Comment:
The last few problems share many common traits and be created with a single function.